tests/components/androidtv/test_media_player.py

```diff
@@ -571,12 +561,12 @@ def _setup(hass, config):
     """Test that setup succeeds with a duplicated config entry."""
     patch_key = "server"
 
-    with patchers.PATCH_ADB_DEVICE_TCP, patchers.patch_connect(True)[
-        patch_key
-    ], patchers.patch_shell("")[patch_key]:
-        assert await async_setup_component(hass, DOMAIN, CONFIG_ANDROIDTV_ADB_SERVER)
+    with patchers.PATCH_ADB_DEVICE_TCP, patchers.patch_connect(
+            True)[patch_key], patchers.patch_shell("")[patch_key]:
+        assert await async_setup_component(hass, DOMAIN,
+                                           CONFIG_ANDROIDTV_ADB_SERVER)
 
-    with patchers.PATCH_ADB_DEVICE_TCP, patchers.patch_connect(True)[
-        patch_key
-    ], patchers.patch_shell("")[patch_key]:
-        assert await async_setup_component(hass, DOMAIN, CONFIG_ANDROIDTV_ADB_SERVER)
+    with patchers.PATCH_ADB_DEVICE_TCP, patchers.patch_connect(
+            True)[patch_key], patchers.patch_shell("")[patch_key]:
+        assert await async_setup_component(hass, DOMAIN,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=4678217)